src/Cms/Roles.php

```diff
@@ -46,7 +47,8 @@ public function canBeChanged()
      * roles that can be created by the
      * current user
      *
-     * @return self
+     * @return \Kirby\Toolkit\Collection
```

Comment:
I don't think these changes are quite right (also in other places). The method returns `$this`, so not just any collection, but `Kirby\Cms\Roles`. Or what was the reason for this change?

Reply:
Because `$this->filter()` method in `canBeChanged`, returns `\Kirby\Toolkit\Collection`, am I wrong?

Reply:
There may be some wrong ones, but I made these changes by checking them

Reply:
I am not a 100% sure.
But I would have thought that the `filter` method (from Kirby\Toolkit\Collection) gets inherited way down to Kirby\Cms\Roles. And the filter method clones the object itself returns it. So within Kirby\Cms\Roles, I would have thought that it clones Kirby\Cms\Roles

Reply:
Hmm, I understand better now. Yes you are right. My bad. I'll check all 👍